programmers/난이도별/level01.예산/HyeonJeong.py

```diff
@@ -0,0 +1,12 @@
+def solution(d, budget):
+    tries = 0
+    for n in sorted(d):
+        if budget >= n:
+            budget -= n
+            tries += 1
```

Comment:
```suggestion
    for n in sorted(d):
        if budget < n: break
        budget -= n
        tries += 1
```
이렇게 할 경우 반복 횟수를 단축할 수 있답니다!